clusterloader2/pkg/measurement/common/api_responsiveness.go

```diff
@@ -201,12 +202,12 @@ func getMetrics(c clientset.Interface) (string, error) {
 }
 
 type apiCall struct {
-	Resource    string                        `json:"resource"`
-	Subresource string                        `json:"subresource"`
-	Verb        string                        `json:"verb"`
-	Scope       string                        `json:"scope"`
-	Latency     measurementutil.LatencyMetric `json:"latency"`
-	Count       int                           `json:"count"`
+	Resource    string                `json:"resource"`
+	Subresource string                `json:"subresource"`
+	Verb        string                `json:"verb"`
+	Scope       string                `json:"scope"`
+	Latency     metrics.LatencyMetric `json:"latency"`
```

Comment:
Can you please:
1. move the "file movement" to a separate PR (it's polluting this one)
2. in this PR, comment on which files are copied (almost copied) from k8s repo and which are written from scratch?

Reply:
1. Done.
2. Comments added. Almost all of the `/util` code is copied.
I tried to split code into gatherers, models and component accessing categories - that's why there are 5 files even though originally were 2.